js/data/locale/mt/list.json

```diff
@@ -17,6 +17,20 @@
             "end": "{0}, u {1}"
         }
     },
+    "or": {
+        "medium": {
+            "2": "{0} jew {1}",
+            "end": "{0}, jew {1}"
```

Comment:
This special case needs to go into the cldr tool that generates the list formats (genlist.js) as a hard-coded format. That way, when we rerun the list format tool, we don't overwrite this special case.